src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public int getSide() {
+        return side;
+    }
+
+    public void setSide(int side) {
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) side * side;

```

Comment:
To avoid type casting you can use double for properties of each type of figures. Check all places